docs/bitcoin-core-usage.md

```diff
@@ -0,0 +1,344 @@
+We will be fetching keys at the BIP 84 default.
+
+```
+$ ./hwi.py -f 8038ecd9 getkeypool --bech32 --keypool 0 100
+[{"desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/0/*)", "internal": false, "range": [0, 100], "timestamp": "now", "keypool": true}]
+```
+
+We now createa a new Bitcoin Core wallet and import the keys into Bitcoin Core. The output is formatted properly for Bitcoin Core so it can be copy and pasted.
+
+```
+$ ../bitcoin/src/bitcoin-cli createwallet "coldcard" true
+{
+  "name": "coldcard",
+  "warning": ""
+}
+$ ../bitcoin/src/bitcoin-cli -rpcwallet=coldcard importmulti '[{"desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/0/*)", "internal": false, "range": [0, 100], "timestamp": "now", "keypool": true}]
+'
+[
+  {
+    "success": true
+  }
+]
+```
+
+Now we repeat the `getkeypool` and `importmulti` steps but set a `--internal` flag and use the change keypath (`m/44h/0h/0h/1`) in `getkeypool` to generate change keys.
+
+```
+$ ./hwi.py -f 8038ecd9 getkeypool --bech32 --keypool --internal 0 100
+[{"internal": true, "timestamp": "now", "desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/1/*)", "keypool": true, "range": [0, 100]}]
+$ ../bitcoin/src/bitcoin-cli -rpcwallet=coldcard importmulti '[{"internal": true, "timestamp": "now", "desc": "wpkh([8038ecd9/84h/0h/0h]xpub6DR4rqx16YnCcfwFqgwvJdKiWrjDRzqxYTY44aoyHwZDSeSB5n2tqt42aYr9qPKhSKUdftPdTjhHrKKD6WGKVbuyhMvGH76VyKKZubg8o4P/1/*)", "keypool": true, "range": [0, 100]}]
+'
+[
+  {
+    "success": true
+  }
+]
+```
+
+
+The Bitcoin Core wallet is now setup to watch a two hundred keys (100 normal, 100 change) from your hardware wallet and you can use it to track your balances and create transactions. The transactions will need to be signed through HWI.
+
+## Usage
+
+Usage of this primarily involves Bitcoin Core. This may be possible to do through the GUI (once all of the keys are imported) but this guide will only cover the command line.
+
+Note: The following was done in a regtest environment. For the purposes of this guide, the `--regtest`s and `--testnet`s that were necessary have been removed and addresses converted to mainnet format.
+
+### Receiving
+
+From the folder containing `bitcoin` and `HWI`, go into `bitcoin`. We will be doing most of the commands here.
+
+```
+$ cd bitcoin
+```
+
+To get a new address, use `getnewaddress` as you normally would
+
+```
+$ src/bitcoin-cli -rpcwallet=coldcard getnewaddress
+3LsU2FHhtasF8V9RT3tfZnEMUExF7JBrME
+```
+
+This address belongs to your hardware wallet. You can check this by doing `getaddressinfo`:
+
+```
+$ src/bitcoin-cli -rpcwallet=coldcard getaddressinfo 3LsU2FHhtasF8V9RT3tfZnEMUExF7JBrME
```

Comment:
Nit: use bech32 for consistency, and to avoid the confusing `embedded` section.

Reply:
Fixed